blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/astyanax/AstyanaxWriter.java

```diff
@@ -134,7 +148,12 @@ public void insertFull(Collection<? extends IMetric> metrics) throws ConnectionE
                     AstyanaxWriter.setLocatorCurrent(locator);
                 }
 
+                if (!isString && !isBoolean && mutationBatch != null) {
```

Comment:
The logic to update `metric_delayed_locator` is same as the one above which is for `metric_locator`. Didnt change much there. I felt like string and boolean metrics arent rolled up. Thats why they are not updated in metrics_locator. I am not 100% sure if that is the intention.